Resources/Prototypes/_DeadSpace/Loadouts/Jobs/Command/officer_blue_shield.yml

```
@@ -2,4 +2,4 @@
 - type: loadout
   id: BlueShieldOfficerJumpsuit
   equipment:
-    jumpsuit: ClothingUniformJumpsuitBlueShield
+    jumpsuit: ClothingUniformJumpsuitKGBMale
```

```suggestion
    jumpsuit: ClothingUniformJumpsuitKGBMale

- type: loadout
  id: BlueShieldOfficerJumpskirt
  equipment:
    jumpsuit: ClothingUniformJumpsuitKGBFemale
```
И в `Resources\Prototypes\_DeadSpace\Loadouts\loadout_groups.yml` в BlueShieldOfficerJumpsuit добавть рядом с BlueShieldOfficerJumpsuit этот BlueShieldOfficerJumpskirt